Library/Homebrew/test/spec_helper.rb

```diff
@@ -198,6 +198,11 @@ def find_files
         CoreTap.instance.path/"formula_renames.json",
         *Pathname.glob("#{HOMEBREW_CELLAR}/*/"),
       ]
+      if OS.mac?
```

Comment:
This doesn't need to be conditional on `OS.mac?` because it's `FileUtils.rm_rf` and will not fail regardless of existence.  As a result, can just inline it below `HOMEBREW_PREFIX/"Caskroom",`

Reply:
good call: 1ad0add